frontend/src/components/form/base/EColorField.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <EParseField
+    ref="input"
+    :value="value"
+    :format="format"
+    :parse="parse"
+    :serialize="serialize"
+    :deserialize="deserialize"
+    :required="required"
+    :vee-id="veeId"
+    :vee-rules="veeRules"
+    reset-on-blur
+    v-bind="$attrs"
+    v-on="$listeners"
+    @input="$emit('input', $event)"
+  >
+    <!-- passing through all slots -->
```

Comment:
Why here again all slots?

Reply:
To not introduce too much difference. The move to specific slots could still be done globally.